src/main/webapp/assets/js/feedScript.js

```diff
@@ -1018,8 +1121,14 @@ window.onclick = function(event) {
 };
 
 document.addEventListener('keydown', function(e) {
-  const isTabPressed = e.key === 'Tab' || e.keyCode === 9;
+  const isSpacePressed = e.keyCode === 32;
+  if (isSpacePressed && document.activeElement === modalUploadLabel) {
+    modalFileUpload.click();
+  }
+});
 
+document.addEventListener('keydown', function(e) {
+  const isTabPressed = e.key === 'Tab' || e.keyCode === 9;
   if (!isTabPressed) {
```

Comment:
why separate these into two functions if they're both responding to the same event?

Reply:
+1

Reply:
I think it makes the code more readable. Are there any drawbacks to separating them?

Reply:
+1 to Melanie's point.

Reply:
Good point, changed.